lib/app/routes/docs.rb

```diff
@@ -0,0 +1,19 @@
+module Xapi
+  module Routes
+    class Docs < Core
+      get '/docs/:track' do |id|
+        track = Xapi::Config.find(id)
+        if track.is_a?(NullTrack)
```

Comment:
Rather than switch on type here, let's give null tracks a `docs` that is unavailable.